benchmarks/nemo/tracer_advection/compute_in_subroutine/Makefile

```diff
@@ -0,0 +1,105 @@
+# Makefile for the version of the tra_adv benchmark that has had
```

Comment:
I think there should be a README in this directory, as you have for the original version.

Reply:
Yes, added.